### ISSUE TITLE
Unable to show plain list using showPicker API

### ISSUE THREAD

**Issue by varungupta85** [aakashns/react-native-dialogs#90]
I am using the `showPicker` API to show a list of things. I want to show them in a plain list without checkboxes or radio buttons but `showPicker` show the list with checkboxes when I don't specify a `type` or specify the `type` as `DialogAndroid.listPlain`. I used the example in the README also

```javascript
const { selectedItem } = await DialogAndroid.showPicker('Pick a fruit', null, {
    items: [
        { label:'Apple', id:'apple' },
        { label:'Orange', id:'orange' },
        { label:'Pear', id:'pear' }
    ]
});
```

and it rendered the below dialog box:
![screen shot 2018-07-24 at 11 57 35 pm](https://user-images.githubusercontent.com/7475133/43158480-61a59e66-8f9d-11e8-8098-c3338c088626.png)

**Comment by Noitidart** [comment]
I will take a look at this in about 8 hours.

**Comment by varungupta85** [comment]
I am using `"react-native": "0.55.4"` and `"react-native-dialogs": "^1.0.1"`. Are you setting any defaults?

**Comment by Noitidart** [comment]
Interesting. No, I'm not setting any defaults.

**Comment by varungupta85** [comment]
I am actually migrating from the old version `0.0.21` to the new API because I have been observing many aborts in the `react-native-dialogs` package. I have migrated everything else except the picker. Could you please post the complete code of the component that you used?

**Comment by Noitidart** [comment]
Excuse my delay:


```
import DialogAndroid from 'react-native-dialogs'

class CommentSort extends PureComponent<Props> {
    render() {
        return (
            <TouchableOpacity onPress={this.handlePress}>
                <Icon style={styles.titleRightIcon} name="sort" />
            </TouchableOpacity>
        )
    }

    handlePress = async () => {
        const { selectedItem } = await DialogAndroid.showPicker('Pick a fruit', null, {
            items: [
                { label:'Apple', id:'apple' },
                { label:'Orange', id:'orange' },
                { label:'Pear', id:'pear' }
            ]
        });
    }
}

export default CommentSort
```

**Comment by varungupta85** [comment]
Never mind the delay, I want to thank you for your prompt responses. I think there was a problem with updating the `react-native-dialogs` package. I deleted the package and reinstalled it and now it is working fine. Not sure, what got messed up earlier but I just removed and installed the package again and it is working as expected now. After hearing from you that you were able to see it fine with the same, that was the only thing I could think of doing. I also wanted to thank you for your efforts in overhauling the package.
